lib/greenhouse_io/api/client.rb

```diff
@@ -99,37 +100,38 @@ def path_id(id = nil)
     end
 
     def permitted_options(options)
-      options.select { |key, value| PERMITTED_OPTIONS.include? key }
+      options.select { |key, _value| PERMITTED_OPTIONS.include? key }
     end
 
     def get_from_harvest_api(url, options = {})
       response = get_response(url, {
-        :query => permitted_options(options),
-        :basic_auth => basic_auth
-      })
-
-      set_headers_info(response.headers)
+                                query: permitted_options(options),
```

Comment:
Bad indentation

Reply:
Not sure why my IDE did this. I'm using the same indentation as in the platform, but I will check this.